syncserver/src/web/handlers.rs

```diff
@@ -32,6 +33,15 @@ pub async fn get_collections(
     db_pool: DbTransactionPool,
     request: HttpRequest,
 ) -> Result<HttpResponse, ApiError> {
+    let _hashed_fxa_uid = String::from(&meta.user_id.hashed_fxa_uid);
```

Comment:
This line likely changing anyway but if a copy's actually needed `.clone()` is more idiomatic